### DISCUSSION TITLE
Can I add a markdown editor on the edit page for the specific model?

### DISCUSSION THREAD

**Discussion by FrancineHuang** [aminalaee/sqladmin#760] [Q&A] [answered]
Hi, I have already read the doc from [working with templates](https://aminalaee.dev/sqladmin/working_with_templates/), I copied all of the templates to another specific path and I have already override it.
also I custom the jinja2 on `create.html` and `edit.html` to add a markdown editor in it, but when I reload the page, nothing changed.

So, is it possible to add a markdown editor for customize? if it was possible, could you give me an example to show it how to do?

**Comment by aminalaee** [comment, marked as answer]
https://github.com/aminalaee/sqladmin/blob/main/docs/cookbook/using_wysiwyg.md
